src/views/Homepage/Questions/texts.tsx

```diff
@@ -37,6 +37,16 @@ const TEXTS: Record<string, any> = {
     answer7_2: "購買指南",
     question8: "如何出售我的 Traveloggers?",
     answer8: "可以透過 OpenSea 出售。",
+    question9: "如何使用航行日誌？",
+    answer9_1: "Traveloggers 擁有者也同時擁有、並可以寫入Logbook。",
+    answer9_2:
+      "第一代 Logbook 直接保存在以太坊區塊鏈，因此在寫入日誌時，需要繳交給以太坊礦工費（Gas Fee）。在寫入日誌前，" +
+      "你可以看到當時的 Gas Fee 以及與字數相對應的費用估算。",
+    answer9_3: "Traveloggers 的開發團隊會提供部分 Gas Fee 補助，支持前 100 名 Logbook 的使用者。同時，繼續尋找費用更低的合適空間，開拓第二代 Logbook。",
+    question10: " 什麼是 Gas Fee？為什麼有 Gas Fee？",
+    answer10_1:
+      "Gas Fee 指的是在區塊鏈上發起一筆交易時，需要負擔的手續費。Gas Fee 是支付給礦工的手續費，當你在以太坊區塊鏈上進行轉賬時，" +
+      "礦工要把你的交易打包並放上區塊鏈，才能使交易完成，在這過程中會消耗區塊鏈的運算資源，所以要支付費用。",
```

Comment:
jsx is usable, I see some cases might need to be bold, or color highlighted like:

```jsx
      <>
         <span color="purple">Gas Fee 指的是在區塊鏈上發起一筆交易時</span>，需要負擔的手續費。<b>Gas Fee</b> 是支付給礦工的手續費，當你在以太坊區塊鏈上進行轉賬時，
         礦工要把你的交易打包並放上區塊鏈，才能使交易完成，在這過程中會消耗區塊鏈的運算資源，所以要支付費用。
      </>
```

Reply:
I noted it last time you pointed out. In fact, I took the same coding pattern in the early Traveloggers page development. But I found out some drawbacks using that way:

- It's not easy to fully understand html structure since elements are separated into different files. I assume a component should clearly describe itself in its file. Thus, I try to keep pure texts in `texts.tsx` in that I don't need to worry there are elements I might overlook.
- Similar reason to styling. `index.moudle.css` should styling nodes written in `index.tsx` only.
- It's easier to extract texts if we need to make a list of page copies for other team members to work.

Some points from me, and love to build a better coding pattern together. 🦞